integration/test_dtda.in

```diff
@@ -8,8 +8,18 @@
 (FAILURE :reason DRUG_NOT_FOUND)
 (FIND-DISEASE-TARGETS :disease NIL)
 (FAILURE :reason INVALID_DISEASE)
+
```

Comment:
It looks like these extra lines were added by accident - if that's the case could you revert the changes to this file?